bin/get_official_heroes.py

```diff
@@ -24,7 +24,7 @@ def get_heroes_data():
 
 def main():
     heroes = get_heroes_data()
-    heroes = [(h['name'], h['slug']) for h in heroes]
+    heroes = [(h['name'], h['slug']) for h in heroes if h['inFreeHeroRotation'] == True]
```

Comment:
Can we instead print all heroes, also printing their free rotation availability?

`heroes = [(h['name'], h['slug'], h['inFreeHeroRotation']) for h in heroes]`